src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -974,8 +1005,14 @@ bool BTypeVisitor::VisitCallExpr(CallExpr *Call) {
     if (!Decl) return true;
 
     string text;
-
-    std::string probe = check_bpf_probe_read_user(Decl->getName());
+ 
+    bool overlap_addr = false;
+    std::string probe = check_bpf_probe_read_user(Decl->getName(),
+                          overlap_addr);
+    if (overlap_addr) {
+      error(GET_BEGINLOC(Call), "bpf_probe_read_user not found. Use latest kernel");
+      return false;
+    }
     if (probe != "") {
       vector<string> probe_args;
 
```

Comment:
if (!rewriter_.isRewritable(GET_BEGINLOC(Call))) {
       return false;
      }

Reply:
Adding this check here,  resolves the test_probe_read3 failure in test_clang.py.